src/js/media-airplay-button.ts

```diff
@@ -66,7 +79,7 @@ class MediaAirplayButton extends MediaChromeButton {
   }
 
   constructor(options: { slotTemplate?: HTMLTemplateElement } = {}) {
-    super({ slotTemplate, ...options });
+    super({ slotTemplate, tooltipContent, ...options });
```

Comment:
(non-blocking) I think we should move away from passing args to the constructor in general.
reason being in the DOM when creating a CE there is no such a thing e.g. `document.createElement('x-button')`. attrs or props are always added after construction. cc @cjpillsbury @heff